src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -973,9 +973,15 @@ void grpc_chttp2_add_incoming_goaway(grpc_chttp2_transport* t,
   }
```

Comment:
Can we delete this empty if whilst here?

Reply:
done. Thanks!